community/kernel/src/main/java/org/neo4j/kernel/impl/api/index/BatchingMultipleIndexPopulator.java

```diff
@@ -334,6 +344,8 @@ private static String allStackTraces()
      */
     private class BatchingIndexPopulation extends IndexPopulation
     {
+        boolean failed;
```

Comment:
i think this variable need to be volatile since addApplicable will be invoked from multiple threads